### ISSUE TITLE
Cleanup codeowners

### ISSUE THREAD

**Issue by derberg** [asyncapi/cli#1287]
tl;dr I'm stepping down as maintainer. It's too big project for me to maintain it regularly.

I propose @Amzani and @Shurtu-gal to join as maintainers.

@Amzani came up with create improvement plan for CLI and will also migrate `server-api` project here. He also leads bounty issues for CLI. Great leader that needs to take ownership on all that next steps, including onboarding of potential maintainers that will move here to CLI (being initially maintainers of `server-api`)

@Shurtu-gal knows the project, contributes now to important DX bounty issue. He also knows CLI a lot as he rewrote our official GitHub Action from Generator based to CLI based. And he will probably move it to cli repo anyway -> https://github.com/asyncapi/github-action-for-cli

With above people joining, I can step down.
@Souvikns stays and will be there to support with some questions related to legacy decisions.

@magicmatatjahu I think we should remove you from codeowners here. We did not use your expertise here for very long (contrary to `asyncapi-react` project. And since the project will go through facelifting I suggest you step down with me, like a king 😆 

---
Let us have it here for at least 2 weeks unless we get all interested parties sharing their view

**Comment by derberg** [comment]
Ok, we just need Maciej, but I know he is out on holidays till the end of March, so we need to wait 😄 

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOGR6MjlxYWVmNjh3d25sd2xuenJiaWtjdWU3czU4YXNrMGNucXA4aSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/g01ZnwAUvutuK8GIQn/giphy-downsized.gif)
